tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -98,10 +103,12 @@ def create_repo_and_code_mapping(
             project_id=self.project.id,
             stack_root=stack_root,
             source_root=source_root,
+            default_branch=default_branch,
```

Comment:
This should have been in place since the beginning. It became obvious when writing tests in #89859